command: /usr/bin/python /u/pedronis/PyPy/dist/pypy/tool/alarm.py 900 /u/pedronis/PyPy/dist/pypy/interpreter/py.py --oldstyle /u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_strptime.py
run by: pedronis@ratthing-b246
sys.platform: linux2
sys.version_info: (2, 3, 5, 'final', 0)
startdate: Thu Apr 28 14:19:19 2005
pypy-revision: 11549
============================================================
faking <type 'unicode'>
faking <type 'module'>
faking <type 'file'>
faking <type 'posix.stat_result'>
faking <type 'posix.statvfs_result'>
faking <type '_sre.SRE_Pattern'>
faking <type 'function'>
faking <type 'time.struct_time'>
faking <type 'instancemethod'>
faking <type '_sre.SRE_Match'>
Traceback (application-level):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_strptime.py", line 506 in ?
    test_main()
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_strptime.py", line 501 in test_main
        CacheTests
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_support.py", line 262 in run_unittest
    run_suite(suite, testclass)
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_support.py", line 247 in run_suite
        raise TestFailed(err)
TestFailed: Traceback (most recent call last):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_strptime.py", line 194, in test_compile
    "Match failed with '%s' regex and '%s' string" %
AttributeError: 'SRE_Pattern' object has no attribute 'pattern'

test_basic (__main__.getlang_Tests) ... ok
test_am_pm (__main__.LocaleTime_Tests) ... ok
test_by_hand_input (__main__.LocaleTime_Tests) ... ok
test_date_time (__main__.LocaleTime_Tests) ... ok
test_lang (__main__.LocaleTime_Tests) ... ok
test_month (__main__.LocaleTime_Tests) ... ok
test_timezone (__main__.LocaleTime_Tests) ... ok
test_unknowntimezone (__main__.LocaleTime_Tests) ... ok
test_weekday (__main__.LocaleTime_Tests) ... ok
test_blankpattern (__main__.TimeRETests) ... ok
test_compile (__main__.TimeRETests) ... ERROR
test_getitem (__main__.TimeRETests) ... ok
test_matching_with_escapes (__main__.TimeRETests) ... ok
test_pattern (__main__.TimeRETests) ... ok
test_pattern_escaping (__main__.TimeRETests) ... ok
test_TypeError (__main__.StrptimeTests) ... ok
test_bad_timezone (__main__.StrptimeTests) ... ok
test_caseinsensitive (__main__.StrptimeTests) ... ok
test_date (__main__.StrptimeTests) ... ok
test_date_time (__main__.StrptimeTests) ... ok
test_day (__main__.StrptimeTests) ... ok
test_defaults (__main__.StrptimeTests) ... ok
test_escaping (__main__.StrptimeTests) ... ok
test_hour (__main__.StrptimeTests) ... ok
test_julian (__main__.StrptimeTests) ... ok
test_minute (__main__.StrptimeTests) ... ok
test_month (__main__.StrptimeTests) ... ok
test_percent (__main__.StrptimeTests) ... ok
test_second (__main__.StrptimeTests) ... ok
test_time (__main__.StrptimeTests) ... ok
test_timezone (__main__.StrptimeTests) ... ok
test_unconverteddata (__main__.StrptimeTests) ... ok
test_weekday (__main__.StrptimeTests) ... ok
test_year (__main__.StrptimeTests) ... ok
test_twelve_noon_midnight (__main__.Strptime12AMPMTests) ... ok
test_all_julian_days (__main__.JulianTests) ... ok
test_day_of_week_calculation (__main__.CalculationTests) ... ok
test_gregorian_calculation (__main__.CalculationTests) ... ok
test_julian_calculation (__main__.CalculationTests) ... ok
test_new_localetime (__main__.CacheTests) ... ok
test_regex_cleanup (__main__.CacheTests) ... ok
test_time_re_recreation (__main__.CacheTests) ... ok

======================================================================
ERROR: test_compile (__main__.TimeRETests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/u/pedronis/PyPy/dist/lib-python-2.3.4/test/test_strptime.py", line 194, in test_compile
    "Match failed with '%s' regex and '%s' string" %
AttributeError: 'SRE_Pattern' object has no attribute 'pattern'

----------------------------------------------------------------------
Ran 42 tests in 728.549s

FAILED (errors=1)
========================== closed ==========================
execution time: 746.773476124 seconds
exit status: 1
